test/metabase/driver/mysql_test.clj

```diff
@@ -339,13 +337,8 @@
   (mt/test-driver :mysql
     (if (System/getenv "MB_MYSQL_SSL_TEST_SSL_CERT")
       (testing "MySQL with SSL connectivity using PEM certificate"
-        (let [e      env/env
-              key-fn (fn [k]
-                       (keyword (str/replace-first (name k) "mb-mysql-ssl-test" "mb-mysql-test")))
-              new-e  (m/map-keys key-fn e)]
-          (with-redefs [env/env new-e]
-            ;; with the mysql-test vars having been set to the mysql-ssl variants, run some test to verify connectivity
-            (metabase.query-processor-test.string-extracts-test/test-breakout))))
+        (tu/with-env-keys-renamed-by #(str/replace-first % "mb-mysql-ssl-test" "mb-mysql-test")
```

Comment:
You should put an alias to `with-env-keys-renamed-by` in `metabase.test` and refer to it by `mt/with-env-keys-renamed-by`.

`metabase.test.util` is an ok place for it to actually live, but test utils should be consumed thru `metabase.test`.

Before we had `metabase.test`, there were way too many test util namespaces and it made writing tests hard because you had to remember which util function was in each. So `metabase.test` provides almost all of the common test utils in one place for easy access.

I don't want to undo all that hard work and have people start having to use `metabase.test.util` directly again.

Reply:
Fixed in both tests.